comit/src/network/protocols/orderbook.rs

```diff
@@ -29,8 +30,10 @@ use std::{
 
 pub use self::order::*;
 
-/// String representing the BTC/DAI trading pair.
-const BTC_DAI: &str = "BTC/DAI";
+lazy_static::lazy_static! {
+    /// The Topic used to publish BTC/DAI orders.
+    static ref BTC_DAI_TOPIC: Topic = Topic::new("BTC/DAI".to_string());
```

Comment:
So, I guess I should start a mailing list discussion about the topic?

Reply:
As I've said in other places, the whole gossipsub needs rethinking/redesigning IMNSHO. Please do hit the mailing list, I'm interested in you view, I have some ideas also.